README.rst

```diff
@@ -1,88 +1,236 @@
-platform-plugin-teams
-#############################
+Platform Plugin Teams
+#####################
 
-.. note::
+|ci-badge| |license-badge| |status-badge|
 
-  This README was auto-generated. Maintainer: please review its contents and
-  update all relevant sections. Instructions to you are marked with
-  "PLACEHOLDER" or "TODO". Update or remove those sections, and remove this
-  note when you are done.
-
-|pypi-badge| |ci-badge| |codecov-badge| |doc-badge| |pyversions-badge|
-|license-badge| |status-badge|
 
 Purpose
 *******
 
-Open edX plugin that includes a custom teams API
+Open edX plugin that includes a custom teams API, which extends the
+functionality of the Open edX default teams API. This plugin includes endpoints
+to create, delete, and list topics, and to add users (members) to teams.
```

Comment:
Can we add in what capacity is the team's API extended?

Reply:
Done